dbt/adapters/databricks/impl.py

```diff
@@ -67,3 +80,113 @@ def execute(
         finally:
             if staging_table is not None:
                 self.drop_relation(staging_table)
+
+    def list_relations_without_caching(
+        self, schema_relation: DatabricksRelation
+    ) -> List[DatabricksRelation]:
+        kwargs = {"schema_relation": schema_relation}
+        try:
+            results = self.execute_macro(LIST_RELATIONS_MACRO_NAME, kwargs=kwargs)
+        except dbt.exceptions.RuntimeException as e:
+            errmsg = getattr(e, "msg", "")
+            if f"Database '{schema_relation}' not found" in errmsg:
+                return []
+            else:
+                description = "Error while retrieving information about"
+                logger.debug(f"{description} {schema_relation}: {e.msg}")
+                return []
+
+        relations = []
+        for row in results:
+            if len(row) != 4:
+                raise dbt.exceptions.RuntimeException(
+                    f'Invalid value from "show table extended ...", '
+                    f"got {len(row)} values, expected 4"
+                )
+            _schema, name, _, information = row
+            rel_type = RelationType.View if "Type: VIEW" in information else RelationType.Table
+            is_delta = "Provider: delta" in information
+            is_hudi = "Provider: hudi" in information
+            relation = self.Relation.create(
+                database=schema_relation.database,
+                schema=_schema,
+                identifier=name,
+                type=rel_type,
+                information=information,
+                is_delta=is_delta,
+                is_hudi=is_hudi,
+            )
+            relations.append(relation)
+
+        return relations
+
+    def get_relation(
```

Comment:
Seems like we don't need to override this method?
https://github.com/dbt-labs/dbt-spark/blob/bc9fc0baaacce28706d0542391704d9ea971cee4/dbt/adapters/spark/impl.py#L162-L168